src/pretix/static/pretixcontrol/js/ui/dragndroplist.js

```diff
@@ -1,48 +1,46 @@
 /*global $, Sortable*/
 $(function () {
     $("[data-dnd-url]").each(function(){
-        var container = $(this),
+        const container = $(this),
             url = container.data("dnd-url"),
             handle = $('<span class="btn btn-default btn-sm dnd-sort-handle"><i class="fa fa-arrows"></i></span>');
 
         container.find(".dnd-container").append(handle);
-        if (container.find("[data-dnd-id]").length < 2) {
+        container.find(".sortable-up, .sortable-down").addClass("sr-only");
+        if (container.find("[data-dnd-id]").length < 2 && !container.data("dnd-group")) {
             handle.addClass("disabled");
             return;
         }
-
+        let didSort = false, lastClick = 0;
+        function handleMouseUp() {
+            if (Date.now() - lastClick < 6000) {
+                container.find(".sortable-up, .sortable-down").removeClass("sr-only");
+            }
+            lastClick = Date.now();
+        }
+        container.find(".dnd-sort-handle").on("mouseup", handleMouseUp);
```

Comment:
This behaves a bit strange IMHO. When I click a drag-handle, but do not drag far enough (ie. outside of the button), it is registered as a click/mouseup. When I then click any other button inside that group, decide not to drag, I get the sortable buttons. I think the 6s delay is to long, maybe 2s might be enough? And it should either apply to all drag-buttons together or (IMHO better) to each button individually. Also, when a successfull drag was done, I think the whole handleMouseUp-Listener could be removed (although that would also mean to remove the help-text/title regarding double-click on that drag-element – so better just leave it connected)?

Also maybe call the method as what it does instead of when its called, so something like `checkIfSortButtonsNeeded`?

Reply:
habs das verhalten nochmal ein bisschen getuned :) 

ich habs auf "alle buttons merken sich die zeit gemeinsam" geändert weil das einfach zu implementieren ist und ich finde es macht nicht so einen großen unterschied, aber die zeit reduziert.

Reply:
Super! Mir würde fast noch weniger Zeit (z.B. 800ms) reichen, aber jetzt triggered es nicht mehr so schnell (und dadurch gefühlt fast zufällig).